### ISSUE TITLE
word file showing content that cannot be checked for accessibility issues because of the current file type

### ISSUE THREAD

**Issue by umairali** [PHPOffice/PHPWord#1378]
I created word file but it's showing some compatibility issue and showing message

> Content that cannot be checked for accessibility issues because of the current file type

![capture](https://user-images.githubusercontent.com/5318005/40406488-f499031e-5e71-11e8-82ed-4c8fb78ff37a.JPG)

Below is my phpword code

```
$phpWord = new \PhpOffice\PhpWord\PhpWord();
$section = $phpWord->addSection();
\PhpOffice\PhpWord\Settings::setOutputEscapingEnabled(true);
\PhpOffice\PhpWord\Shared\Html::addHtml($section, $html, true, false);
$objWriter = \PhpOffice\PhpWord\IOFactory::createWriter($phpWord, 'Word2007');
$filePath = $_SERVER['DOCUMENT_ROOT'] . $filenameWithExt;
$objWriter->save($filePath);
```

**Comment by troosan** [comment]
does adding the follwoing help?
```php
		Settings::setCompatibility(true);
```

**Comment by umairali** [comment]
No showing same message

**Comment by umairali** [comment]
I am using like this

```
\PhpOffice\PhpWord\Settings::setOutputEscapingEnabled(true);
 \PhpOffice\PhpWord\Settings::setCompatibility(true);
 \PhpOffice\PhpWord\Shared\Html::addHtml($section, $html, true, false);
```

**Comment by umairali** [comment]
I am using table width 100% in style and I try 600px i think the issue is table width in html. Can you please guide me how can i apply style from php code on all tables of html.

**Comment by umairali** [comment]
ms word show this message on click on convert

![capture55](https://user-images.githubusercontent.com/5318005/40417127-1172074c-5e90-11e8-84e5-f5a81fd615d1.JPG)

**Comment by umairali** [comment]
the issue is file version is old that why it's showing this compatibility issue

**Comment by umairali** [comment]
add this line 
`$phpWord->getCompatibility()->setOoxmlVersion(15);`

before 
`\PhpOffice\PhpWord\Shared\Html::addHtml($section, $html, true, false);`


[](http://phpword.readthedocs.io/en/latest/recipes.html#remove-compatibility-mode-text-in-the-ms-word-title-bar)
